source/scripting_v2/GTA/GameVersion.cs

```diff
@@ -69,5 +69,7 @@ public enum GameVersion
 		VER_1_0_1737_0_NOSTEAM,
 		VER_1_0_1737_6_STEAM,
 		VER_1_0_1737_6_NOSTEAM,
+        VER_1_0_1868_0_STEAM,
```

Comment:
Tabs please =)

Reply:
`.editorconfig` would be useful to prevent this in the future.

Reply:
There is one already, so I don't know how this happened: https://github.com/crosire/scripthookvdotnet/blob/master/.editorconfig

Reply:
Didn't saw it. Maybe he used a Code Editor/IDE that does not supports `.editorconfig`

Reply:
What is this sorcery of which you speak !

Reply:
Fixed that in https://github.com/crosire/scripthookvdotnet/pull/901/commits/540251ec0a61278d9ce987e841c07ed467153d0b. Make sure you don't have the "Hide whitespace changes" option selected in GitHub. But I can also see it in my local Git client.